### ISSUE TITLE
Missing mods in forge-1.14.4-28.1.0

### ISSUE THREAD

**Issue by wrenger** [ocpu/Boxlin#6]
I have problems getting forge running with boxlin.
I'm using the forge-1.14.4-28.1.0 example mod template with the modifications from your [README](https://github.com/ocpu/Boxlin/tree/v3#installation).
However, Forge is unable to load the mod and shows the message: 
```The Mod File /.../mc-nbt/tool/build/resources/main has mods that were not found```

Did I miss something during the migration?

Here is corresponding commit: [GitLab MC NBT Tool](https://gitlab.com/l4r0x/mc-nbt-tool/commit/0d7e5dd0a51f6504e861f27129942506a89f5fc6)
And the Logfile: [latest.log](https://github.com/ocpu/Boxlin/files/3816580/latest.log)

**Comment by ocpu** [comment]
I have looked into your problem and what I have found is that you do not include the kotlin gradle tool that compiles your kotlin code.

```diff
 buildscript {
     repositories {
         jcenter()
         mavenCentral()
         maven { url = 'https://files.minecraftforge.net/maven' }
     }
     dependencies {
         classpath group: 'net.minecraftforge.gradle', name: 'ForgeGradle', version: '3.+', changing: true
+        classpath group: 'org.jetbrains.kotlin', name: 'kotlin-gradle-plugin', version: '1.3.41'
     }
 }

 apply plugin: 'net.minecraftforge.gradle'
 // Only edit below this line, the above code adds and enables the necessary things for Forge to be setup.
 apply plugin: 'eclipse'
+apply plugin: 'kotlin'
 apply plugin: 'maven-publish'
```

What I also found (it might just be me) is that I had to clean the generated files and re-open my editor (IntelliJ IDEA) in that same directory. The generated files where in `build/ .gradle/ .idea/ run/`.

**Comment by wrenger** [comment]
Oh yes I missed that, thanks.
Now it works as expected.

I also had to convert all occurrences of `FMLJavaModLoadingContext` to `BoxlinContext` and the `@Mod.EventBusSubscriber` annotations doesn't work on `object` classes. I have to register them manually with `MinecraftForge.EVENT_BUS.register(this)`. 
Or is there a better solution?

**Comment by ocpu** [comment]
Thank you for alerting me to this problem I should have fixed this with 9a96eb9 and `3.0.1`. So that you can use `@Mod.EventBusSubscriber` annotations again.

**Comment by wrenger** [comment]
Thank you. This helps a lot.

Unfortunately I encountered another problem: 
When I try to load the mod apart from the development environment in a normal forge installation (1.14.4-28.1.0) the FML shows the following Error.
```
Mod File boxlin-3.0.0.jar needs language provider boxlin:3 or above to load
We have found none
```
I only installed my mod and your language provider.
Here is the corresponding logfile: [latest.log](https://github.com/ocpu/Boxlin/files/3825802/latest.log)
And the latest build of my mod: [GitLab CI](https://gitlab.com/l4r0x/mc-nbt-tool/-/jobs/artifacts/master/download?job=build)

I have no experience with the new FML procedure, so maybe I have forgotten something?
